common/app/views/fragments/omnitureScript.scala.html

```diff
@@ -1,21 +1,12 @@
 @(item: Option[model.MetaData])
 
 @import conf.Static
-@import conf.Configuration
 @import views.support.OmnitureAnalyticsAccount
 
 @item.map{ item =>
     <script>
         @* must always be set before the Omniture file is parsed *@
         window.s_account = '@OmnitureAnalyticsAccount(item)';
+        @Html(Static.js.omnitureJs)
     </script>
 }
-
-<script>
```

Comment:
`s_account` is referenced in the omniture vendor js file that we are inlining through `omnitureJs`. Is there any possible way it could be referenced before our assignment?